### PR TITLE
RUN-9 Create Profile Picture

### PR DESCRIPTION
**Pull request by kelvincklo**
## Changes

> Created profilePicture component. Also created Components folder.

## Checklist

- [x] Pull request has descriptive context useful to reviewer. Screenshots are attached as necessary.
- [x] Pull request title matches Linear ticket.
- [x] Ticket has been claimed on Linear
- [x] Pull Request has been assigned to yourself
- [ ] Pull request has been approved by CodeOwner
- [ ] Pull request has been approved by one other student

### CODE REVIEW

**Review by NeoLou — CHANGES_REQUESTED** [review]
I think you should try to test it first to see if it works as intended by importing it into App.js and seeing if it renders when you start the app.

**Comment by kelvincklo** [comment]
> I think you should try to test it first to see if it works as intended by importing it into App.js and seeing if it renders when you start the app.

Ty for the comment Neo, I've imported the ProfilePicture component and tested it in App.js.

**Review by Jnwong116 — APPROVED** [review]

**Review by NeoLou — APPROVED** [review]
LGTM!

File: client/src/component/profilePicture/profilePicture.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+import "./profilePicture.css";
+
+class ProfilePicture extends React.Component {
+  render() {
+    const { img_link } = this.props;
+
+    return <img class="profile-pic" src={img_link} />;
```
Comment (Jnwong116) [inline client/src/component/profilePicture/profilePicture.js]: For future reference:
Might need to change how we load up the src depending on where we get the images from. 

`src={require("../../assets/images/" + this.props.icon)}`

**Pull request merged**